controller/internal/supervisor/iptablesctrl/iptables.go

```diff
@@ -335,6 +335,7 @@ func (i *iptables) UpdateRules(version int, contextID string, containerInfo *pol
 	// Install all the new rules. The hooks to the new chains are appended
 	// and do not take effect yet.
 	if err := i.installRules(newCfg, containerInfo); err != nil {
+		zap.L().Error("unable to install rules on update", zap.Error(err))
```

Comment:
we should return an error here. I am not sure why it doesn't ... I see this is a bug